ext/ConstructionBaseLinearAlgebraExt.jl

```diff
@@ -0,0 +1,87 @@
+module ConstructionBaseLinearAlgebraExt
+
+if isdefined(Base, :get_extension)
+    using ConstructionBase
+    using LinearAlgebra: LinearAlgebra, SubArray, LinRange, Cholesky, LowerTriangular, UpperTriangular, Tridiagonal
+else
+    using ..ConstructionBase
+    using ..LinearAlgebra: LinearAlgebra, SubArray, LinRange, Cholesky, LowerTriangular, UpperTriangular, Tridiagonal
+end
+
+### SubArray
```

Comment:
SubArray isnt in LinearAlgebra, neither is ReshapedArray, PermutedDimsArray, or LinRange...

Reply:
Oops, sorry.

Please feel free to edit directly to this PR. I almost posted an issue instead but I thought I might take a quick attempt at kicking off this PR.

Reply:
This is fine, it just needs those bits back where they were. But I will leave that to you ;)